customcommands/assets/customcommands-database.html

```diff
@@ -20,6 +20,25 @@ <h2>Custom Commands Database</h2>
 </header>
 {{template "cp_alerts" .}}
 
+<div>
+    <section class="card">
+        <header class="card-header">
+            <h2 class="card-title">Database Usage - {{.DatabaseUsagePercent}}% Full</h2>
+        </header>
+        <div class="card-body">
+            <div class="col-auto">
+                <h4>{{.TotalDatabaseUsage}} Entries used of total {{.TotalDatabaseCapacity}}</h4>
+                <p>
+                    Total calculated by member count multiplied by 50{{if .IsGuildPremium}}0{{end}} (<code>{{.ActiveGuild.MemberCount}}*50{{if .IsGuildPremium}}0{{end}}</code>).{{if not .IsGuildPremium}} Get 10 times the
+                    database space with <a href="/premium">premium</a>!{{end}}
```

Comment:
I'm torn on plugging premium here—ultimately Des's call to make, but personally I'm not a big fan of this on the front-end.

Reply:
It's plugged for the custom commands check, I think it's beneficial to know that its possible to increase the size.